File: Programmers/PRGM_소수만들기.py
import math

def solution(nums):
    
    def isprime(num) :
        for i in range(2, int(math.sqrt(num))+1) :
            if num % i == 0 :
                return False
        return True
    
    l = len(nums)
    number = 0
    cnt = 0
    for i in range(l-2) :
        for j in range(i+1, l-1) :
            for k in range(j+1, l) :
                number = nums[i] + nums[j] + nums[k]
                if isprime(number) :
                    cnt += 1
                    
    return cnt
                
# print(solution([1,2,7,6,4]))    

File: Programmers/PRGM_소수만들기.js
function solution(nums) {
  function isprime(number) {
    for (let i = 2; i <= Number(Math.sqrt(number)); i++) {
      if (number % i === 0) {
        return false;
      }
    }
    return true;
  }

  var l = nums.length;
  var number = 0;
  var cnt = 0;

  for (let i = 0; i <= l - 3; i++) {
    for (let j = i + 1; j <= l - 2; j++) {
      for (let k = j + 1; k < l; k++) {
        number = nums[i] + nums[j] + nums[k];
        if (isprime(number) === true) {
          cnt += 1;
        }
      }
    }
  }
  return cnt;

  return answer;
}
